libs/basics/src/errors.ts

```diff
@@ -4,3 +4,15 @@
 export function extractErrorMessage(error: unknown): string {
   return error instanceof Error ? error.message : String(error);
 }
+
+/**
+ * Determines whether an error is a Node fs module non-existent file or directory error
+ */
+export function isNonExistentFileOrDirectoryError(error: unknown): boolean {
```

Comment:
Arguably this belongs in `@votingworks/backend` since the frontends use `/basics` as well. Since it happens to work in that environment I don't have a strong preference, but something to consider.

Reply:
Good point! I'm leaning toward keeping the helper in `libs/basics` because `libs/backend` depends on other modules that could potentially use this helper, e.g. `libs/auth`, `libs/db`, `libs/logging`, `libs/usb-drive`. If we moved the helper into `libs/backend`, we'd introduce a circular dependency by having those other modules pull from `libs/backend`